docker/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: "3.7"
+
+services:
+  web:
+    image: mempool/frontend:latest
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    command: "./wait-for db:3306 --timeout=720 -- nginx -g 'daemon off;'"
+    ports:
+      - 80:8080
+    environment:
+      FRONTEND_HTTP_PORT: "8080"
+      BACKEND_MAINNET_HTTP_HOST: "api"
+  api:
+    image: mempool/backend:latest
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    command: "./wait-for-it.sh db:3306 --timeout=720 --strict -- ./start.sh"
+    volumes:
+      - ./data:/backend/cache
+    environment:
+      RPC_HOST: "127.0.0.1"
+      RPC_PORT: "8332"
+      RPC_USER: "mempool"
+      RPC_PASS: "mempool"
+      ELECTRUM_HOST: "127.0.0.1"
+      ELECTRUM_PORT: "50002"
```

Comment:
It seems this assumes you already have romanz/electrs running on the host machine. Maybe we should add romanz/electrs to this docker-compose template, perhaps as a fragment option?

Reply:
thanks for the feed back - i will add this.

Reply:
We also want to avoid conflicts with other bundled docker implementations (like umbrel) so it will be an optional command in make - i will add some notes in documentation as well.